.tool-versions

```diff
@@ -1,2 +1,2 @@
-erlang 25.3.2-1
+erlang 25.3.2.3
```

Comment:
is there any reason for this change, the OTP whose version with a `-` is our fork which for clustering by the rlog method

Reply:
@lafirest Sorry, my mistake, I'll roll it back.